api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- <databaseChangeLog
+		xmlns="http://www.liquibase.org/xml/ns/dbchangelog/2.0"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/2.0
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd"> -->
+
+<databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd
+    http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+
+	<changeSet id="1.0" author="Partners In Health">
```

Comment:
Can you also make the changeset ids to either be uuids or of a pattern like "idgen_identifier_source_table_create"?

Reply:
Should this pattern apply to other changesets as well, such as `idgen_auto_generation_option_add_foreign-key_constraint`?

Reply:
The constraints should be contained in their table creation changesets